packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -288,6 +290,22 @@ class IconButton extends StatelessWidget {
   ///  * [Feedback] for providing platform-specific feedback to certain actions.
   final bool enableFeedback;
 
+  /// Box constraints for the button.
```

Comment:
Optional size constraints for the button.

Reply:
Edited.